src/Raven.Studio/typescript/viewmodels/common/showDataDialog.ts

```diff
@@ -1,40 +1,55 @@
 import dialog = require("plugins/dialog");
 import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
 import copyToClipboard = require("common/copyToClipboard");
+import aceEditorBindingHandler = require("common/bindingHelpers/aceEditorBindingHandler");
 
 class showDataDialog extends dialogViewModelBase {
 
-    constructor(private title: string, private inputData: string, elementToFocusOnDismissal?: string) {
+    width = ko.observable<string>("");
+    inputData = ko.observable<string>();
+
+    constructor(private title: string, inputData: string, elementToFocusOnDismissal?: string, calcWidth: boolean = false) {
         super(elementToFocusOnDismissal);
+
+        this.inputData(inputData);
+        if (calcWidth) {
+            this.width(this.calcWidth(inputData) + "px");
+        }  
+    }
+
+    calcWidth(code: string): number {
+        const arrayOfLines = code.match(/[^\r\n]+/g);
+        const maxLineLength = Math.max.apply(Math, $.map(arrayOfLines, el => el.length));
```

Comment:
.map is standard javascript function, you don't have to use jquery for that:

Additionally lodash has max:
https://lodash.com/docs/4.17.4#max

So you can replace this with:

`
_.max(arrayOfLines.map(el => el.length))
`